src/utils/fileUtils.ts

```diff
@@ -0,0 +1,76 @@
+import * as path from "path";
+import { TextDocument, window, workspace, WorkspaceFolder } from "vscode";
+
+/**
+ * Return the current workspace uri from root
+ *
+ * @param currentFile the opened TextDocument
+ * @returns the path from root of the current workspace
+ */
+export function getWorkspaceUri(currentFile: TextDocument): string {
```

Comment:
Rename currentFile with document and do that for other function which have textdocument